lib/v2/reuters/radar.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    'reuters.com': {
+        _name: '路透社',
+        '.': [
+            {
+                title: '路透导航',
+                docs: 'https://docs.rsshub.app/traditional-media.html#lu-tou-she',
+                source: [':navigation/:category?', '/'],
+                target: '/reuters/:navigation/:category?',
+            },
+            {
+                title: '订阅作者',
+                docs: 'https://docs.rsshub.app/traditional-media.html#lu-tou-she-ding-yue-zuo-zhe',
+                source: ['/authors/:category?', '/'],
+                target: '/reuters/authors/:category?',
+            },
+            {
+                title: '订阅深度调查栏目',
+                docs: 'https://docs.rsshub.app/traditional-media.html#lu-tou-she-shen-du-diao-cha-lan-mu',
```

Comment:
```suggestion
                title: '导航',
                docs: 'https://docs.rsshub.app/traditional-media.html#lu-tou-she',
                source: ['/:navigation/:category?', '/'],
                target: '/reuters/:navigation/:category?',
            },
            {
                title: '订阅作者',
                docs: 'https://docs.rsshub.app/traditional-media.html#lu-tou-she',
                source: ['/authors/:category?', '/'],
                target: '/reuters/authors/:category?',
            },
            {
                title: '深度调查栏目',
                docs: 'https://docs.rsshub.app/traditional-media.html#lu-tou-she',
```
`title` should match the H3 header
all `docs` should point to the H2 header